wagit/headers/GitConfig.h

```diff
@@ -0,0 +1,40 @@
+//
+// Created by Ranveer Aggarwal on 28/08/20.
+//
+
+#include <filesystem>
+
+#define MINI_CASE_SENSITIVE
+#include "../includes/ini.h"
+
+#ifndef WAGIT_GITCONFIG_H
+#define WAGIT_GITCONFIG_H
+namespace WAGit
+{
+    namespace FS = std::filesystem;
+
+    using Path = FS::path;
+
+    /*!
+     * The repository's config
+     */
+    class GitConfig
+    {
+        long version = 0;
+        bool fileMode = false;
+        bool bare = false;
```

Comment:
Move to the end of the class? Like you did in the repository class?

Reply:
Thanks! Done.